tests/frame/test-create.py

```diff
@@ -1305,6 +1305,12 @@ def test_create_from_numpy_floats_mixed(numpy):
     assert DT.to_list() == [[4.0, 3.5, None, 9.33]]
 
 
+def test_create_from_numpy_strings(np):
+    src = ["alpha", "beta", "gamma"]
+    DT = dt.Frame([np.str_(s) for s in src])
+    assert_equals(DT, dt.Frame(src))
+
+
```

Comment:
I would also add a test for `np.str()` even though it seems to work fine atm.

Reply:
That's way too unlikely. Currently `np.str` is the same as builtin `str`. What could they change it into? Not into a wrapper class for strings, because they already have `np.str_` for that. They could remove it I guess, or convert it into a submodule, and then your proposed test would fail. But there is no need for us to even know that such change will have happened.

Reply:
Well, when we write a test we normally don't know in advance what would cause it to fail.

Reply:
A test is only useful when its failure would signal that we need to fix something in datatable.
Whereas for this test its failure will only cause us to delete the test.

Reply:
It depends on why it would fail.